### ISSUE TITLE
F/A-18 coordinates as precise

### ISSUE THREAD

**Issue by MacFlorent** [the-paid-actor/dcs-dtc#82]
Opening this to notify that I'm working a modification of your application to allow for the input and cockpit push of "precise" coordinates in the Hornet. I had done this already in an old version of DTC.

If you are interested I will submit a PR when done.
If you are already working on it I will stop because there is no need to do the same thing twice :)

Thanks.

**Comment by MacFlorent** [comment]
It's done in my fork and will be tested by my team.

**Comment by MacFlorent** [comment]
Still FYI.

I did another iteration, to allow for input of coordinates in any format that can be parsed by CoordinateSharp. This addresses #73 I think (only for the F/A-18).

This was more code that I would have liked though, and is getting a bit away of from the coding concepts of DTC. For example, I did not use the DTCTextbox or the DTC Coordinate classes, and reworked all coordinates logics to segregate the display from the storage. So maybe it's a bit much for a PR, I would not want to step on your developments concepts.

I will still submit it when it will have been tested, and let you decide if you wish to keep it.
